src/python/setup.py

```diff
@@ -40,6 +40,6 @@
 	ext_modules = cythonize([
 		Extension("seccomp", ["seccomp.pyx"],
 			# unable to handle libtool libraries directly
-			extra_objects=["../.libs/libseccomp.a"]),
+			extra_objects=["../.libs/libseccomp.so"]),
```

Comment:
```suggestion
			libraries=["seccomp"],
```
Works for me over at https://github.com/conda-forge/seccomp-split-feedstock/blob/a96cea714e957f04451de86a4aa34ded5e90a8d7/recipe/0001-Dynamic-linking-for-Python-package.patch .
(But I only ever tried it with our build system.)

Reply:
For this, I do install `libseccomp` beforehand separately, of course. If you'd want to use `libraries=...` before installing, you'd have to pass on the library search path accordingly.

Reply:
E.g.,
```suggestion
			library_dirs=["../.libs"],
			libraries=["seccomp"],
```

Reply:
Yep, makes sense and looks better.